src/SimplCommerce.WebHost/appsettings.json

```diff
@@ -1,41 +0,0 @@
-﻿{
```

Comment:
We shouldn't commit the change in ConnectionStrings, it's specific to developers. We recommend using secret manager http://docs.simplcommerce.com/en/latest/development-tips/